src/Compatibility/Core/src/iOS/Compatibility.iOS.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Condition="'$(EnvironmentBuildPropsImported)' != 'True'" Project="..\..\..\..\..\eng\Environment.Build.props" />
   <PropertyGroup>
     <Description>iOS Compatibility Backend for Microsoft.Maui</Description>
-    <AssemblyName>Microsoft.Maui.Controls.Compatibility</AssemblyName>
+    <AssemblyName>Microsoft.Maui.Controls.Compatibility.iOS</AssemblyName>
```

Comment:
MSBUILD can't handle this project and the Android one having the same assembly name. It gets confused. Since we aren't packaging these versions of the dlls this is hopefully fine and doesn't ultimately lead to weird confusing scenarios